identity/src/main/java/com/android/identity/CredentialData.java

```diff
@@ -1472,8 +1472,12 @@ Pair<PrivateKey, byte[]> selectAuthenticationKeyHelper(boolean allowUsingExhaust
 
         // We've found the key with lowest use count... so if this is beyond maximum uses
         // so are all the other ones. So fail if we're not allowed to use exhausted keys.
-        if (candidate.mUseCount >= mAuthMaxUsesPerKey && !allowUsingExhaustedKeys) {
-            return null;
+        if (candidate.mUseCount >= mAuthMaxUsesPerKey) {
+            if (!allowUsingExhaustedKeys) {
+                return null;
+            } else {
+                Log.w(TAG, "Using exhausted key.");
+            }
```

Comment:
Unnecessary ` else { .. }`, also I think `i` is a more appropriate level than `w`.